_delphi_utils_python/tests/validator/test_datafetcher.py

```diff
@@ -24,20 +28,32 @@ def test_make_date_filter(self):
     # Solution from https://stackoverflow.com/questions/15753390/
     #how-can-i-mock-requests-and-the-response
     def mocked_requests_get(*args, **kwargs):
+        # TODO #1863: convert to requests_mock
         class MockResponse:
             def __init__(self, json_data, status_code):
                 self.json_data = json_data
                 self.status_code = status_code
 
             def json(self):
                 return self.json_data
+
+            def raise_for_status(self):
+                if self.status_code != 200:
+                    raise HTTPError()
         if len(kwargs) == 0:
             return MockResponse([{'source': 'chng', 'db_source': 'chng'},
                 {'source': 'covid-act-now', 'db_source': 'covid-act-now'}], 200)
         elif kwargs["params"] == {'signal': 'chng:inactive'}:
             return MockResponse([{"signals": [{"active": False}]}], 200)
         else:
             return MockResponse([{"signals": [{"active": True}]}], 200)
+
+    @requests_mock.Mocker(kw="mock_requests")
```

Comment:
```suggestion

    # the `kw` approach is needed here because otherwise pytest thinks the 
    # requests_mock arg is supposed to be a fixture
    @requests_mock.Mocker(kw="mock_requests")
```